exercises/06-deploy-to-cf.md

```diff
@@ -32,56 +44,65 @@ This is what the service instances should look like in the SAP BTP cockpit:
 ![SCP Services](images/scp-services.png)
 
 ## Adjust the manifest.yml file
+
 Let's adjust the [manifest.yml](../manifest.yml), the configuration file for your Cloud Foundry application.
-Take a look at the services section, where we reference the previously created services instances.
-Replace all occurrences of `<participantId>` in the manifest.yml file.
+Take a look at the services section, where we reference the previously created service instances.
 
 ## Build and push your application
+
 Run the following to build and package your application:
+
 ```sh
 npm run ci-build && npm run ci-package
 ```
 
 Eventually, push it to Cloud Foundry:
+
 ```sh
 cf push
 ```
-As this will take a moment, proceed to the next step in the mean time.
+
+As this will take a moment, proceed to the next step in the meantime.
 
 ## Configure Destinations on SAP BTP
-Login to the [SAP BTP cockpit](https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trialhome) and find your subaccount. Check the *Connectivity* tab on the left. Here you will find your configuration for connectivity services: **destinations** and **cloud connectors**.
-You will need to [configure a cloud connector](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/e6c7616abb5710148cfcf3e75d96d596.html) when you want to connect to an on premise system. Today, we will connect to a mocked cloud system and will therefore skip this step.
+
+Login to the [SAP BTP cockpit](https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trialhome) and find your subaccount. Check the _Connectivity_ tab on the left. Here you will find your configuration for connectivity services: **destinations** and **cloud connectors**.
+You will need to [configure a cloud connector](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/e6c7616abb5710148cfcf3e75d96d596.html) when you want to connect to an on-premise system. 
 
 As we will retrieve data from SAP S/4HANA and SAP SuccessFactors, we have to configure those systems as destinations.
 Click on the destinations and add two new destinations:
 
 SAP S/4HANA destination:
+
 > ```
 > Name: S4HANA
 > Type: HTTP
-> URL: https://codejam-s4-mock-server.cfapps.eu10.hana.ondemand.com/
+> URL: <S4-server-url>
 > Proxy type: Internet
-> Authentication: NoAuthentication
->```
+> Authentication: <S4-authentication-type>
+> ```
 
 SAP SuccessFactors destination:
+
 > ```
 > Name: SFSF
 > Type: HTTP
-> URL: https://codejam-sfsf-mock-server.cfapps.eu10.hana.ondemand.com/
+> URL: <SFSF-server-url>
 > Proxy type: Internet
-> Authentication: NoAuthentication
->```
+> Authentication: <SFSF-authentication-type>
+> ```
+
+Please refer [.env](../.env) file if you don't remember the `URL` and `Authentication` as you defined them in [exercise 1](01-getting-started.md#setup-the-destinations-environment-variable).
 
 This is what the configured destinations should look like in the SAP BTP cockpit:
 ![SCP Destinations](images/scp-destinations.png)
```

Comment:
[req] I think that the `scp-destinations.png` shows also the service instances and not the destiantions.

Reply:
I replaced correct image